### DISCUSSION TITLE
[Feature request] Write a Post for Github Sponsors (like as Patreon)

### DISCUSSION THREAD

**Discussion by ptkdev** [community/community#23265] [New to GitHub]
<p>Github sponsors is an excellent platform.</p>
<p>I believe it lacks a really important feature: write and limit the posts to tiers.</p>
<p>As a patreon, it is helpful to provide posts and attachments for your circle of sponsors. It is useful for giving them prizes, previews or beta and could increase the desire to donate for a project knowing that you have “previews” which can be beta versions,previews, screenshots or even assets.</p>
<p>This feature could make me leave patreon forever and make me use github sponsors only.</p>
<p>P.s. I can’t find a thread / conversations in the github community for “sponsors” and I wrote here. If it is bad, please moderator move this post to the correct section.</p>

**Comment by MChevy422** [comment, marked as answer]
<p>Hello <span><a class="mention" href="/ptkdev">@ptkdev</a></span>,  <br><br>Thank you for your feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.<br><br>Once again, thank you for your input!<br><br>Greatly appreciated<br><br>MChevy422</p>
